### DISCUSSION TITLE
stupid question, but is there a way to change colors, icons and background etc

### DISCUSSION THREAD

**Discussion by toppits** [fooyin/fooyin#276] [Q&A] [answered]
My installation is a flatpak, which seems to be the only available. Does that mean the theme is not changeable beyond the 3 preset themes?

**Comment by Kri-U** [comment, marked as answer]
On v6.2 and later you can change the colors & fonts yourself in Edit > Settings > Interface > Colours & Fonts.
Here you can change colors & fonts and saves them as themes
for info I'm on OpenSUSE TW with Fooyin v6.2 installed via the OpenSUSE repositories, but I'm pretty sure it's the same on the Flatpak version

![image](https://github.com/user-attachments/assets/c7942644-ecfe-4e30-b2c8-8320460918cf)
